### ISSUE TITLE
Increase maximum valid zoom

### ISSUE THREAD

**Issue by Hordeking** [tumic0/GPXSee#244]
The code currently specifies 19 is the maximum possible zoom, even when the max is set higher in the xml file. According to what I can tell, at least the USGS maps have the capability of going all the way to level 24 (not able to confirm why these aren't actually available yet). So hard-coding a max of 19 seems a bit pre-mature, though as a default max for being unspecified map sets is probably reasonable.

- So basically I'm requesting that zoom levels up to or even better than 24 are allowed if defined as such in the xml.

**Comment by tumic0** [comment]
Makes sense. I will let the user to configure the max zoom to an arbitrary value (A link to some map that really implements such high zooms would be helpful).
